src/main/java/uk/gov/companieshouse/service/AppealService.java

```diff
@@ -83,44 +84,59 @@ private void createContactInChips(Appeal appeal, String userId) {
     }
 
     protected ChipsContact buildChipsContact(Appeal appeal) {
-
         final String companyNumber = appeal.getPenaltyIdentifier().getCompanyNumber();
         final OtherReason otherReason = appeal.getReason().getOther();
+        final IllnessReason illnessReason = appeal.getReason().getIllnessReason();
 
         final ChipsContact chipsContact = new ChipsContact();
         chipsContact.setCompanyNumber(companyNumber);
         chipsContact.setDateReceived(appeal.getCreatedAt().format(DATE_TIME_FORMATTER));
+        ReasonType reasonType = appeal.getReason().getReasonType();
 
-        final String contactDescription = "Appeal submitted" +
+        String contactDescription = "Appeal submitted" +
             "\n\nYour reference number is your company number " + companyNumber +
             "\n\nCompany Number: " + companyNumber +
             "\nEmail address: " + appeal.getCreatedBy().getEmailAddress() +
-            "\n\nAppeal Reason" +
-            "\nReason: " + otherReason.getTitle() +
-            "\nFurther information: " + otherReason.getDescription() +
-            "\nSupporting documents: " + getAttachmentsStr(appeal.getId(), otherReason);
-
+            "\n\nAppeal Reason";
+
+        List<Attachment> attachmentList;
+
+        switch(reasonType.getReasonType()){
+            case ReasonType.OTHER:
+                attachmentList = otherReason.getAttachments();
+                contactDescription +=
+                    ("\nReason: " + otherReason.getTitle() + "\nFurther information: " + otherReason.getDescription());
+                break;
+            case ReasonType.ILLNESS:
+                attachmentList = illnessReason.getAttachments();
+                contactDescription += ("\nIll Person " + illnessReason.getIllPerson() +
+                "\nOther Person: " + illnessReason.getOtherPerson() +
+                    "\nIllness Start Date: " + illnessReason.getIllnessStart() +
+                    "\nContinued Illness" + illnessReason.getContinuedIllness() +
+                    "\nIllness End Date: " + illnessReason.getIllnessEnd() +
```

Comment:
Illness End Date can be empty, we should add a default string, probably `None`

Reply:
I believe this is a null issue as the entry wouldn't be there on the mongo DB or the contact so that would suffice in practice.